.github/workflows/ci.yml

```diff
@@ -47,11 +46,9 @@ jobs:
         with:
           cache: true
       - name: Install dependencies
-        run: |
-          (cd packages/dart && flutter pub get)
-          (cd packages/flutter && flutter pub get)
+        run: flutter pub get
       - name: Analyze code
-        run: flutter analyze packages/flutter --no-fatal-infos
+        run: flutter analyze / --no-fatal-infos
```

Comment:
you can delete this as well `check-code-analysis-flutter`

Reply:
This job was deleted